requirements_tools/upgrade_requirements.py

```diff
@@ -114,6 +119,7 @@ def pip_install(*argv):
             '--index-url', args.index_url,
             '--exec-count', str(args.exec_count),
             '--exec-limit', str(args.exec_limit),
+            '--pip-tool', str(args.pip_tool),
```

Comment:
`args.pip_tool` is already a str, the two things above this were integers (thus requiring `str(...)`)